cluster/gce/util.sh

```diff
@@ -1144,7 +1144,7 @@ NODE_PROBLEM_DETECTOR_CUSTOM_FLAGS: $(yaml-quote "${NODE_PROBLEM_DETECTOR_CUSTOM
 CNI_STORAGE_URL_BASE: $(yaml-quote "${CNI_STORAGE_URL_BASE:-}")
 CNI_TAR_PREFIX: $(yaml-quote "${CNI_TAR_PREFIX:-}")
 CNI_VERSION: $(yaml-quote "${CNI_VERSION:-}")
-CNI_SHA1: $(yaml-quote "${CNI_SHA1:-}")
```

Comment:
I'm not positive whether populating CNI_SHA1 is still required or not to support skewed nodes (e.g. a 1.20 API server and 1.18 node) ... @karan, any idea?

Reply:
afaik that should not matter, this script should not depend on the apiserver version